content/blogposts/2024/codyception-coding-cody-tools-with-cody.md

```diff
@@ -0,0 +1,159 @@
+---
+title: "Codyception: Coding Cody Tools With Cody For Coding With Cody"
+authors:
+  - name: Kynlo Akari
+    url: https://twitter.com/Kynlos
+  
+publishDate: 2024-01-29T10:00-07:00
+description: "This post with teach you how Custom Commands in Cody can simplify your coding process by automating repetitive tasks, saving time and effort. Kynlo guide you through how these commands make coding more efficient and manageable, transforming complex tasks into simple, one-click solutions."
+tags: [blog]
+slug: "codyception-coding-cody-tools-with-cody"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/codyception-coding-cody-tools-with-cody-v4.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/codyception-coding-cody-tools-with-cody-v4.png
+---
+
+<Alert type="secondary">This is a guest blog post from our community. <a href="https://discord.com/servers/sourcegraph-969688426372825169" target="_blank">Join our Discord</a> to meet the author and connect with other developers passionate about Code Search and AI!</Alert>
+
+OK, I'll admit it took me longer than it should to write that title without messing up, but here we are, there that is, and here we go!
+
+So, now that I've successfully confused half of you before I've even started, let's talk about Custom Commands with Cody.
+
+---
+
+### Friends, Coders, Netizens, lend me your snippets!
+
+
+To start with, let's delve into the significance of Custom Commands within the realm of programming and automation. These commands play a pivotal role by affording you the ability to define specific actions tailored to your precise needs. In more elementary terms, they act as a time-saving mechanism, sparing you considerable effort, especially in tasks characterized by repetition. Consider scenarios like utilizing Cody to insert specific types of comments, code editing, feature incorporation, or even language translation for entire files.  In even more simplistic terms, they let you make a button that does a thing, then you can click that button to do that thing an unlimited amount of times without having to type anything again!  Sound good? I bet it does...so let's talk about it in the context of our main man Cody.
+
+The elegance of Cody lies in its seamless facilitation of the creation and integration of these commands. This capability transforms the coding landscape, rendering your journey notably more streamlined and efficient. Cody becomes a reliable companion, adept at executing nuanced coding tasks with dexterity.
+
+In essence, Custom Commands serve as the virtuosos of your coding orchestra, orchestrating intricate symphonies with finesse. As you navigate the coding landscape, allow these commands to be your skilful conductors, ensuring a harmonious and efficient coding experience. 
```

Comment:
```suggestion
In essence, Custom Commands serve as the virtuosos of your coding orchestra, ensuring a harmonious and efficient coding experience.
```

I'd move this paragraph and merge it with the previous one, since it was greatly shortened